File: Problem_1.java
public class Problem_1 {

    public static int[] twoSum(int[] nums, int target){
        for(int i = 0; i < nums.length; i++){
            for(int j = i + 1; j < nums.length; j++){
                if (nums[j] == target - nums[i]){
                    return new int[]{i, j};
                }
            }
        }
        return new int[0];
    }

    public static void main(String[] args) {
        int[] nums = {2, 7, 13, 17, 23};
        int target = 30;

        int[] result = twoSum(nums, target);

        System.out.println("Indices of the two numbers that add up to the target: " + result[0] + ", " + result[1]);
    }

}

File: Go/Problem_1.go
package main

import (
	"fmt"
)

func twoSum(nums []int, target int) []int {
	var i, j int

	for i = 0; i < len(nums); i++ {
		for j = i + 1; j < len(nums); j++ {
			if nums[j] == target-nums[i] {
				return []int{i, j}
			}
		}
	}
	return []int{}
}

func execute_problem1() int {
	var array []int = []int{2, 7, 13, 17, 23}
	var target int = 9

	var result []int = twoSum(array, target)

	fmt.Printf("Target was the sum of %d and %d\n", result[0], result[1])

	return 0
}
